bats/tests/extensions/containers.bats

```diff
@@ -22,6 +29,14 @@ encoded_id() { # variant
     wait_for_container_engine
 }
 
+@test 'default to custom docker context' {
+    if ! using_docker; then
+        skip 'docker context only applies when using docker backend'
+    fi
+    docker context create bats-invalid-context --docker 'host=tcp://invalid.test:99999'
```

Comment:
It is a bit more robust to remove a left-over context here instead if relying on `teardown_file` to have cleaned it up before:

```
    run docker context rm bats-invalid-context
    assert_nothing
```